hardware/asic/skywater/Makefile

```diff
@@ -31,16 +32,17 @@ sram: openram_config.py
 	mv temp sram
 #SRAM wrapper generation
 
-iob_dp_ram.v:
+iob_sp_ram_be.v:
 #It was added for test only -will be removed-
 #	~/Documents/iob-mem/software/python/memakerwrap.py $(ASIC_MEM_TECH) iob_dp_ram $(ASIC_DPRAM_TYPE) 0 1 $(SRAM_W) 8 4 1 > $@
-	$(MEM_DIR)/software/python/memakerwrap.py $(ASIC_MEM_TECH) iob_dp_ram $(ASIC_DPRAM_TYPE) 0 1 $(SRAM_W) 8 4 1 > $@
+	$(MEM_DIR)/software/python/memakerwrap.py $(ASIC_MEM_TECH) $(ASIC_MEM_MODULE) $(ASIC_SPRAM_TYPE) 0 1 $(SRAM_W) 8 4 1 > $@
```

Comment:
There is no point in have ASIC_MEM_MODULE variable. You can have it hardcoded as before, it is only used in here.

Reply:
but this variable is only here in this Makefile

Reply:
And boot.hex is there only temporarily for testing only i mentioned it in original message

Reply:
> but this variable is only here in this Makefile

That's why you don't need to have a variable. You can write it hardcoded here.

Reply:
I will do it

Reply:
> And boot.hex is there only temporarily for testing only i mentioned it in original message

I understand that, but boot.hex is a generated file. So you should let the Makefiles generate it automatically and use it as is.